main-node/repeater/default_repeater.py

```diff
@@ -32,6 +32,6 @@ def decision_function(self, history, point, **configuration):
 
             # Calculating average.
             for index, value in enumerate(result):
-                result[index] = eval(self.WSClient._result_data_types[index])(round(value / len(all_experiments), 2))
+                result[index] = eval(self.WSClient.results_data_types[index])(round(value / len(all_experiments), 2))
```

Comment:
It`s a bug.
In HTTP-based WSClient this field not 'protected', in Sockets-based - protected.
Needed to be fixed by renaming this field to protected in legacy HTTP-based WSClient.